cookbooks/redis/recipes/install.rb

```diff
@@ -114,6 +112,10 @@
     action :nothing
   end
 
+  # check if redis is installed
+  # if it is, restart it when redis-server.service changes
+  # without the check, "notifies :restart" fails when redis-server isn't installed yet
+  is_redis_installed = Mixlib::ShellOut.new("[ -e #{redis_bin_path} ]").run_command.exitstatus == 0
```

Comment:
Why can't we use `run_installer`?

Reply:
If redis isn't installed (during the first time the recipe runs), you can't call `notifies :restart, "service[redis-server]", :immediately`. Maybe I can remove `:immediately`?

Reply:
I removed :immediately and removed is_redis_installed.